lucupy/helpers/__init__.py

```diff
@@ -149,8 +244,14 @@ def angular_distance(ra1: float, dec1: float, ra2: float, dec2: float) -> float:
 
 
 def mask_to_barcode(mask: str, inst: Optional[str]) -> str:
```

Comment:
I didn't even know we had this. I'm kind of confused by the logic, but we can look at that later. We should probably o a cleanup of lucupy at some point.